### ISSUE TITLE
`retype build --override` will cause issue in the main page

### ISSUE THREAD

**Issue by vmohir** [retypeapp/retype#336]
When I use `retype build docs --output=public/` or `retype watch docs` everything is fine but when I run `retype build docs --output=public/ --override="{ }"` the output's main page (the index.html) has the default content (only the "Welcome" title) and all the markdown in the `index.md` file at the root of the project is ignored.

Worth mentioning that all these commands work fine:

```
retype build docs --output=public/
retype build docs --override="{ }"
retype build docs
retype build docs --override="{ \"output\": \"../public\"}"
```

**Comment by geoffreymcgill** [comment]
HI @vmohir. Thank you for reporting this defect. Very interesting issue. 🤔 We will investigate and fix right away. Hopefully there is a work around we can find before the fix is available publicly with the next release.

**Comment by vmohir** [comment]
Of course! The workaround is the last command:

```retype build docs --override="{ \"output\": \"../public\"}"```

**Comment by geoffreymcgill** [comment]
Hi @vmohir. This defect has been fixed and the fix will be included in the upcoming `v2.4.0` release of Retype. I don't have an exact date for the release of `v2.4.0`, but it should be very soon.

Once you upgrade to `v2.4.0`, your project with the leading `_` in the folder name will just start working. No changes to your project or the exact `--override` flag will be required.

Hope this helps.
